src/composables/default-ref.ts

```diff
@@ -0,0 +1,22 @@
+import { computed, ref, UnwrapRef } from '@nuxtjs/composition-api'
+
+const NotSet = Symbol('NotSet')
+
+/**
+ * Produces a `computed` that returns a default value until
+ * an explicit value is set.
+ *
+ * @param getDefault - A function returning the default value
+ */
+export const defaultRef = <T>(getDefault: () => T) => {
+  const explicitlySet = ref<T | typeof NotSet>(NotSet)
+  return computed<T>({
+    get() {
+      if (explicitlySet.value !== NotSet) return explicitlySet.value as T
```

Comment:
When reading this condition "not equal to not set" feels like a double negative. Can we invert the `if`?

Reply:
Done in https://github.com/WordPress/openverse-frontend/pull/1129/commits/bd5d8bb8d7568cf93a94ea62316401bef8b180aa